Coalition Yottrite (#4896)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			`	Shortly after landing you're greeted by an Arach wearing what you judge to be some sort of uniform. He takes a translation device out of his pocket and formally greets you. "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
			choice
				`	"Are you the shipwright I'll be taking to Lugglop?"`
					goto servant
				`	"What's with this place? Why are there so many buildings all over?"`
					goto property
			label servant
			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as a chauffeur, I work."`
				goto mistress
			label property
			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arach ships. To the left, a marketplace is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
			label mistress

> 			`	He then asks that you follow him, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must." You follow him along the path, he gives a bit more detail on some of the buildings as you walk near them. What really piques your interest though is that one of the buildings, which he seemed to willingly ignore, had armed guards at the door, with no indication of being members of the Heliarch.`

Reviewer: ```suggestion
			`	He then asks that you follow him, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must." You follow him along the path, and he gives a bit more detail on some of the buildings as you walk near them. What really piques your interest though is that one of the buildings, which he seemed to willingly ignore, had armed guards at the door, with no indication of being members of the Heliarch.`
```

			`	Reaching the entrance of the mansion, it isn't as castle-like as the walls, but it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
			`	He guides you through the mansion. You pass by several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students who came here to study with the shipwrights of House Mallob.`
			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
			choice
				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`

> 				`	"A Kimek? I thought this was an Arach House."`

Reviewer 2: Perhaps have a choice that isn't as joking as these two.

Author: added new one

			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student at the university here, she was."`

> 			`	Finally, the reason why you had to come all the way to this part of <origin> becomes clear; Pichiie mentions she would like to hear your stories about where you first got the yottrite in the first place before boarding your ship to head to <planet>. You tell her all samples you've come across were found in systems where the star had gone supernova. It's a short explanation, but she appears to be satisfied, and after saying her goodbyes to other members of the House, you're both brought to your ship.`

Reviewer 2: Remove repetition of "first" in "first got the yottrite in the first place."

				`	"What discoveries have you made about it?"`
					goto findings
			label findings
			`	"To share our findings with you, allowed we are not," Pichiie says.`
			`	"Forgive us you must, Captain, but told to keep everything a secret, we were," Lugglop continues. "Perhaps after our presentation, public our findings could be made."`
			label accept
			`	They all follow you to your ship after the containers are ready to be loaded.`
			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents, both Kimek, are accompanying him.`
			`	"Here to ensure the secrecy of our discoveries, they are," he says. Seeing as they are only two more passengers, you agree to carry them to <planet>, and they all board your ship.`
				accept
	on complete
		payment 310000
		conversation
			`	Immediately when you land, Lugglop gives you your payment of <payment>. You and the four researchers then follow some Heliarch agents that were waiting for you at the docks. You five are brought to an area akin to a theater with some type of podium on the stage. The seats are all taken by Heliarch agents.`

> 			`	Lugglop's team apparently will wait until the contents of the two containers are brought here, so you're asked to describe the systems with supernovae where you encountered yottrite asteroids. This only lasts a few minutes, as the audience doesn't seem too keen on inquiring about details, though one particular Saryd did puzzle you when they asked if the all the supernovas you visited "were natural occurrences, or artificially produced."`

Reviewer 2: ```suggestion
			`	Lugglop's team apparently will wait until the contents of the two containers are brought here, so you're asked to describe the systems with supernovae where you encountered yottrite asteroids. This only lasts a few minutes, as the audience doesn't seem too keen on inquiring about details, though one particular Saryd did puzzle you when they asked if all the supernovas you visited "were natural occurrences, or artificially produced."`
```

This question from the Saryd is very on the nose. We know from the Wanderer campaign that the Wanderers have their suspicion that the supernovae were artificially induced, but how would the Heliarchs think of that? Do they have some sort of understanding of this?

Author: changed to not be so obvious

> 			`	After you finish, and right as the materials for Lugglop's presentation arrives, you're escorted back to your ship by some agents. Several hours later, you're contacted by Lugglop, who says the prpesentation is finished, and his group is coming back to your ship. When they arrive, they each thank you for providing the means to perform their experiments. "Officially commended for our efforts, we were!" He says.` 

Reviewer: ```suggestion
			`	After you finish, and right as the materials for Lugglop's presentation arrives, you're escorted back to your ship by some agents. Several hours later, you're contacted by Lugglop, who says the presentation is finished, and his group is coming back to your ship. When they arrive, they each thank you for providing the means to perform their experiments. "Officially commended for our efforts, we were!" he says.` 
```
(I think) since this would normally be a comma and you're just using an exclamation mark to clarify his tone, the punctuation should still be treated as if it were a comma (if that makes any sense).

> 			`	"Though to publicly share our findings, allowed we are not still, invited to be part of the team here that will continue the research, we were," Pichiie adds. The Saryd twins say they will reside here for as long as the project continues, while Pichiie comments on how she'll often be taking trips to visit her family in House Mallob's property.`

Reviewer: ```suggestion
			`	"Though allowed to publicly share our findings, we are not still, invited to be part of the team here that will continue the research, we were," Pichiie adds. The Saryd twins say they will reside here for as long as the project continues, while Pichiie comments on how she'll often be taking trips to visit her family in House Mallob's property.`
```
Thought this helped with clarity (personal preference).

			`	"As for me, a job as overseer, I still have to perform," Lugglop says. "Stay here for a few days I will, but even with all the advanced equipment of this ring, still our Coalition's most capable, the smelters of Bloptab's Furnace are. In their optimal state they must be, if to deepen our knowledge on yottrite, we are. Return there shortly, I will."`

> 			`	They thank you again for all your help, and wish you safe travels as you continue exploring.`

Reviewer: ```suggestion
			`	They thank you again for all your help and wish you safe travels as you continue exploring.`
```

	on offer
		set "license: Coalition"
		fail

mission "Coalition Yottrite 1"
	name "More Yottrite"
	description "You were offered great payment for bringing three more samples of the rare mineral yottrite to <destination>."
	minor
	landing
	to offer
		has "Coalition: Contributor: done"
	to complete
		has "Coalition Yottrite 2: offered"
	source

> 		near Bloptab 1 100

Reviewer 2: ```suggestion
		near "Bloptab" 1 100
```

	source "Delve of Bloptab"
	destination "Shadowed Valley"
	on offer
		conversation
			`You wait around the spaceport for some minutes before finally spotting the Arach. He invites you to his office.`
			`	"Responsible for overseeing deliveries of such exotic materials to the warehouses here, I also am. For the inconvenience and bureaucracy, I apologize," he says. "Forget to properly introduce myself, I often do. Lugglop, I am called."`
			choice
				`	"Nice to meet you, I suppose."`
				`	"Do you need more samples? Is that why you called me here?"`
			`	"Harder to keep track of them, it becomes, the more samples there are," he says. "Notified by the Heliarchs, I was. No more samples, for now, I can request. Risk letting them be stolen, they cannot." He then shows you a map of the Coalition, pointing to a system in Saryd space. "Use more delicate procedures and research, to study the samples we must. For that, friends of mine on <destination>, will be waiting," he says. "Bring them here, would you, Captain?"`
			choice
				`	"Of course, they'll be here soon."`
					accept
				`	"Sorry, I'm not one to take passenger jobs."`
					decline
	on complete
		conversation

> 			`When you land on <planet>, Lugglop contacts you and informs you that he has already told his friends about you. He says they will be waiting in the spaceport.`

Reviewer 2: Given how short this is, this should be a dialog.

mission "Coalition Yottrite 5"
	name "Saryd Specialists"
	description "Transport the twins Nasilor and Alituri to <destination>, where they will assist in the Yottrite research."

> 	passengers 2

Reviewer 2: This mission should have a blocked dialog in case the player doesn't have the necessary space.

	description "Head to <destination> to pick up a member of House Mallob so that they may help in trying to apply Yottrite to an alloy."
	landing
	to offer
		has "Coalition Yottrite 5: done"
	source "Delve of Bloptab"
	destination "Market of Gupta"
	on offer
		conversation
			`Lugglop is waiting for you when you land and greets the two researchers as you help them get their things out of the ship.`
			`	He points them to a hotel where he had a room prepared for them, and asks you to follow him to his office again.`
			`	"Filled to the brim with cutting edge equipment, our facilities here and on the neighbor planet are. Alas, venture much into the theoretical field needed to study such rare elements, our personnel here does not."`
			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked some aspects of yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
			`	You're about to head back to your ship when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`
				accept
	on accept

> 		payment 200000

Reviewer 2: You can simply put this payment in the `on offer` since this will always accept.

mission "Coalition Yottrite 7"
	name "Master Shipwright"
	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."

> 	passengers 1

Reviewer 2: Blocked dialog here too.

			`	Reaching the entrance of the mansion, it isn't as castle-like as the walls, but it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
			`	He guides you through the mansion. You pass by several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students who came here to study with the shipwrights of House Mallob.`
			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
			choice
				`	"I take it that's the shipwright?"`
				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
				`	"A Kimek? I thought this was an Arach House."`

> 			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student at the university here, she was."`

Reviewer 2: Now that you have a non-joking choice, it should probably lead into a different line than the joking ones, as there's no need for the butler to ignore my genuine question.

Author: added a line that's the same but different to fit with the non-joke choice

			`	Finally, the reason why you had to come all the way to this part of <origin> becomes clear; Pichiie mentions she would like to hear your stories about where you got the yottrite in the first place before boarding your ship to head to <planet>. You tell her all samples you've come across were found in systems where the star had gone supernova. It's a short explanation, but she appears to be satisfied, and after saying her goodbyes to other members of the House, you're both brought to your ship.`
				accept
	on complete
		event "yottrite research break" 165
		payment 200000
		conversation
			`As with the Saryd twins, Lugglop greets Pichiie and tells her of a hotel he prepared for her time here.`
			`	"Ready, my team is now, Captain," he says, after Pichiie leaves with her luggage. "Enrich and research samples as needed, for the next few months we will. Contact you again, I will, should the need for your services arise once again."`

> 			`	He hands you <payment>, and walks away.`

Reviewer 2: This could probably be a dialog as well.

event "yottrite research break"

mission "Coalition Yottrite 8"
	name "One Last Sample"
	description "Acquire one sample of yottrite and deliver it to <destination> so that Lugglop has some for his presentation to the Coalition's scientific community."
	landing
	to offer
		has "event: yottrite research break"
	source

> 		near Pelubta 1 100

Reviewer 2: ```suggestion
		near "Pelubta" 1 100
```

	destination "Pelubta Station"
	on offer
		conversation
			`When you land, you receive a message from Lugglop, the Arach who you brought several Yottrite samples to.`
			`	"Captain, some major findings, we have. Preparing a presentation about our research, to present in Ring of Wisdom, we are," he says. "Only, run out of pure samples, we have. Bring us one once more, would you?"`
			choice
				`	"Okay, I'll bring you a piece of yottrite once again."`
				`	"Sorry, but I'm not mining that any more."`
					decline
			`	He thanks you and says he and the rest of the team will be waiting in <destination>. "Moved here about halfway through the project, we did, to make use of the station's facilities."`
				accept
	on complete
		outfit "Yottrite" -1
		payment 220000

> 		dialog	`Lugglop pays you <payment> as you deliver the yottrite, accompanied by a Heliarch agent. "Finish preparing our presentation, we now must. If to help us further you wish, in the spaceport meet us you must."`

Reviewer 2: ```suggestion
		dialog `Lugglop pays you <payment> as you deliver the yottrite, accompanied by a Heliarch agent. "Finish preparing our presentation, we now must. If to help us further you wish, in the spaceport meet us you must."`
```

mission "Coalition Yottrite 9"
	name "Yottrite Presentation"
	description "Transport Lugglop and his team, alongside <cargo>, to <destination>, where they will share with the Coalition's scientific community their findings on Yottrite."
	passengers 6

> 	cargo "Yottrite Presentation Materials" 6.14

Reviewer 2: Blocked dialog here too.

	to offer
		has "Coalition Yottrite 8: done"
	source "Pelubta Station"
	destination "Ring of Wisdom"
	on offer
		conversation

> 			`You meet Lugglop in the spaceport and follow him to meet with Nasilor, Alituri, and Pichiie, who are overseeing some workers place a few sealed metal crates inside a container and the yottrite sample you just brought here into another.`
> 			`	They detail how they have prepared all the material they needed and now just need a transport to <destination>.`

Reviewer 2: ```suggestion
			`You meet Lugglop in the spaceport and follow him to meet with Nasilor, Alituri, and Pichiie, who are overseeing some workers place a few sealed metal crates inside a container and the yottrite sample you just brought here into another. They detail how they have prepared all the material they needed and now just need a transport to <destination>.`
```

			`	"If with us, you come, Captain, perhaps a better detailed description of the systems where the yottrite was found, you could provide," Nasilor says.`
			`	"<payment>, I would pay you," Lugglop says.`
			choice
				`	"Of course I'll take you there. Have the containers loaded into my ship."`
					goto accept
				`	"What discoveries have you made about it?"`
					goto findings
			label findings
			`	"To share our findings with you, allowed we are not," Pichiie says.`
			`	"Forgive us you must, Captain, but told to keep everything a secret, we were," Lugglop continues. "Perhaps after our presentation, public our findings could be made."`
			label accept
			`	They all follow you to your ship after the containers are ready to be loaded.`

> 			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents, both Kimek, are accompanying him.`
> 			`	"Here to ensure the secrecy of our discoveries, they are," he says. Seeing as they are only two more passengers, you agree to carry them to <planet>, and they all board your ship.`

Reviewer 2: ```suggestion
			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents, both Kimek, are accompanying him. "Here to ensure the secrecy of our discoveries, they are," he says. Seeing as they are only two more passengers, you agree to carry them to <planet>, and they all board your ship.`
```

				accept
	on complete
		payment 310000
		conversation

> 			`	Immediately when you land, Lugglop gives you your payment of <payment>. You and the four researchers then follow some Heliarch agents that were waiting for you at the docks. You five are brought to an area akin to a theater with some type of podium on the stage. The seats are all taken by Heliarch agents.`

Reviewer 2: ```suggestion
			`Immediately when you land, Lugglop gives you your payment of <payment>. You and the four researchers then follow some Heliarch agents that were waiting for you at the docks. You five are brought to an area akin to a theater with some type of podium on the stage. The seats are all taken by Heliarch agents.`
```

			`	After discussing with his colleagues for a bit, he announces that he is willing to pay you 220,000 credits for the mineral sample.`
			choice
				`	"Sounds like a fair deal to me, sold."`
				`	"Sorry, but I think I'll hold on to it. It was very hard to get it in the first place."`
					decline
			`	He thanks you and transfers the Yottrite from your ship to his own, an Arach Spindle, with the help of some spaceport workers. Right before departing, he hands you the credits, and says that you should bring a few more samples to <destination>, "Three more, you should bring, if interested you are."`
				accept
	on accept
		outfit "Yottrite" -1
		payment 220000

mission "Coalition Yottrite 2"
	name "Even More Yottrite"
	description "You were once again asked to bring several samples of the rare mineral Yottrite to <destination>."
	landing
	to offer
		has "Coalition Yottrite 1: offered"
	to complete
		has "Coalition Yottrite 3: offered"
	source "Bloptab's Furnace"
	on offer
		fail "Coalition Yottrite 1"
		conversation

> 			`Coming down the atmosphere of <origin>, you're contacted by the Arachi you sold some Yottrite to. He asks if you have any more samples to sell him.`

Reviewer 3: ```suggestion
			`Coming down the atmosphere of <origin>, you're contacted by the Arach you sold some Yottrite to. He asks if you have any more samples to sell him.`
```

> 			`Coming down the atmosphere of <origin>, you're contacted by the Arachi you sold some Yottrite to. He asks if you've got any more samples to sell him now.`

Reviewer 4: ```suggestion
			`Coming down the atmosphere of <origin>, you're contacted by the Arachi you sold some Yottrite to. He asks if you've got any more samples to sell him.`
```

			choice
				`	"Not right now, sorry."`
					defer
				`	"I have some, yeah. Where should I bring them to?"`

> 			`	He tells you to land as usual. Once your ship is brought to the underground area, some workers help unload the Yottrite, and you follow them along some areas that were restricted to you other times you came here.`

Author: rephrased the end part

> 			`	He tells you to land as usual. Once your ship is brought to the underground area, some workers help unload the Yottrite, and you follow them into a restricted section of the complex, the guards at the entrance watching closely as your group passes by.`

Author: added few words to do that, changed the structure(?) a bit in doing so, hopefully didn't spawn any new errors in doing that

> 			`	You walk along extremely wide industrial catwalks, overlooking endless smelters and forges, of various sizes. Though the heat of the place might have been influencing your perception, you're almost certain the largest smelters must be around the size of a light warship.`

Reviewer 4: ```suggestion
			`	You walk along extremely wide industrial catwalks, overlooking endless smelters and forges of various sizes. Though the heat of the place might have been influencing your perception, you're almost certain the largest smelters must be around the size of a light warship.`
```

			`	After walking for too long in the uncomfortably hot and stuffy interiors of the underground compound, you and the several tons of Yottrite are finally brought to a large area, similar to a warehouse, but with several workers operating machinery and making sure ore pieces are brought to specific conveyor belts.`

> 			`	As the workers leave you to bring the Yottrite to a separate section of the place, you're greeted by the Arach who contacted you. "In your debt, we are, Captain <last>," he says, handing you 660,000 credits. "Achieved, much will be, if significant progress with the samples we make."`

Author: I'm well aware of that, but again, didn't show up properly when testing in-game, so wrote it down manually.

> 			`	As the workers leave you to bring the Yottrite to a separate section of the place, you're greeted by the Arach who contacted you. "In your debt, we are, Captain <last>," he says, handing you 660,000 credits. "Achieved, much will be, if significant progress with the samples we make."`

Reviewer 5: Strange.

			choice
				`	"What sort of progress?"`
					goto progress
				`	"What are you hoping to achieve?"`
					goto achieve
			label progress
			`	"Non-standard thermodynamic properties, Yottrite has. Absorb heat around them, the crystals do. Difficult to mix it into a compound, it is," he explains. "If how to overcome this obstacle, we learn, many applications, the element could have."`
				goto end
			label achieve
			`	"Run several tests on the Quarg equipment, for centuries, our Coalition has. React similarly to Yottrite, some of their equipment does. A material involved in the making of their outfits, it is," he explains. "If how to do such a thing ourselves, we learn, maybe replicate their technology, or resume construction of the ringworlds, we could."`
			label end
			`	Some worker by the Yottrite crates shouts something, and the Arach signals he will be heading there shortly.`

> 			`	"Once again, if any more Yottrite, you acquire, bring it here, you may," he says before leaving. "Tirelessly researching it, we are. If in a larger batch, you could bring them, progress quicker, we might."`

Author: the `them` in this refers Yottrite samples

> 			`	"Once again, if any more Yottrite, you acquire, bring it here, you may," he says before leaving. "Tirelessly researching it, we are. If in a larger batch, you could bring them, progress quicker, we might."`

Reviewer 5: Misread, my bad.

			`	He says that about 7 samples of the mineral should do for next time, and finally leaves. Some workers come shortly after to escort you back to your ship through the maze of smelters.`
				accept
	on accept
		outfit "Yottrite" -3
		payment 660000

mission "Coalition Yottrite 3"
	name "Neighbor's Mailbox"

> 	description "Deliver the <cargo> to the sister world of<origin>, <destination>."

Reviewer 4: ```suggestion
	description "Deliver the <cargo> to the sister world of <origin>, <destination>."
```

	landing
	cargo "Experimental Yottrite Samples" 5
	to offer
		has "Coalition Yottrite 2: offered"
	source "Bloptab's Furnace"
	destination "Delve of Bloptab"
	on offer
		fail "Coalition Yottrite 2"
		conversation

> 			`Once again, you come to <origin> with a supply of Yottrite, and are contacted by the Arachi here. They ask if you're ready to sell what samples you've brought them now.`

Reviewer 4: ```suggestion
			`Once again, you come to <origin> with a supply of Yottrite and are contacted by the Arachi here. They ask if you're ready to sell what samples you've brought them.`
```

			choice
				`	"Maybe some other time."`
					defer
				`	"Sure am. Have some people wait by the docking area to help me unload the samples."`
			`	As the workers get the ore samples out of your ships, you realize that this time the Arach that asked for your help wasn't the one to contact you, nor was he waiting for you on the ground.`

> 			`	The workers say they will bring the Yottrite to the warehouse on their own, and you're told to wait. When one of them comes back, followed by other workers with different crates, they hand you 1,540,000 credits, and ask for permission to load the crates onto your ship.`

Author: when testing the `<payment>` didn't show up properly if I used it like that, so I wrote it down manually.
changed the rest in a different manner

> 			`	"Roughly 5 tons, they weight. To <destination>, this planet's neighboring world, you must bring them. Requested it so, the boss has," they say.`

Reviewer 4: ```suggestion
			`	"Roughly 5 tons, they weigh. To <destination>, this planet's neighboring world, you must bring them. Requested it so, the boss has," they say.`
```

			`	As you leave your ship, the Arach is waiting for you, alongside some Heliarch officers. They have the cargo removed from your ship, without saying much.`
			`	Right before the Heliarch ships prepare to escort you back to the standard docking area, he says, "In the spaceport, meet you I will."`

mission "Coalition Yottrite 4"
	name "Yottrite Research"
	description "Lugglop, one of the Arachi that oversee the mining operations in Bloptab, asked that you pick up some friends of his on <destination>, so that they may help with the Yottrite research."
	to offer
		has "Coalition Yottrite 3: done"
	source "Delve of Bloptab"
	destination "Shadowed Valley"
	on offer
		conversation
			`You wait around the spaceport for some minutes, to finally spot the Arach, who invites you to his office there.`
			`	"Responsible for overseeing deliveries of such exotic materials to the warehouses here, I also am. For the inconvenience and bureaucracy, I apologize," he says. "Forget to properly introduce myself, I often do. Lugglop, I am called."`
			choice
				`	"Nice to meet you, I suppose."`
				`	"Do you need more samples? Is that why you called me here?"`

> 			`	"The harder to keep track of the samples, it becomes, the more samples there are," he says. "Notified by the Heliarchs, I was. No more samples, for now, I can request. Risk letting them be stolen, they do not want."`

Author: did some changes to the wording

			`	He then shows you a map of the Coalition, pointing to a system in Saryd space.`
			`	"Use more delicate procedures and research, to study the samples we must. For that, friends of mine on <destination>, waiting will be," he says. "Bring them here, would you, Captain?"`
			choice
				`	"Of course, they'll be here soon."`
					accept
				`	"Sorry, I'm not one to take passenger jobs."`
					decline
	on complete
		conversation
			`When you land on <planet>, Lugglop contacts you, and informs you that he has already told his friends about you. He says they will be waiting in the spaceport.`

mission "Coalition Yottrite 5"
	name "Saryd Specialists"
	description "Transport the twins Nasilor and Alituri to <destination>, where they will assist in the Yottrite research."
	passengers 2
	to offer
		has "Coalition Yottrite 4: done"
	source "Shadowed Valley"
	destination "Delve of Bloptab"
	on offer
		conversation

> 			`A Saryd wearing their versions of a lab coat is waiting for you when you enter the spaceport.`

Reviewer 4: ```suggestion
			`A Saryd wearing their version of a lab coat is waiting for you when you enter the spaceport.`
```

			`	"Working with Lugglop, you are?" He asks, and you nod. "Nasilor, my name is. Packing up our belongings, my sister is. Bring you to our laboratory, I shall."`

> 			`	You follow him deep into the underground spaceport, to the "residential" section, which to you is indistinguishable from a research compound. Still on the way, the endless doors to your left and right occasionaly give space for massive glass panels, giving you a nice view of the massive crystal formations here.`

Reviewer 4: ```suggestion
			`	You follow him deep into the underground spaceport, to the "residential" section, which to you is indistinguishable from a research compound. Still on the way, the endless doors to your left and right occasionally give space for massive glass panels, giving you a nice view of the massive crystal formations here.`
```

			`	Nasilor stops by a door immediately after one of these "viewing sites", and opens it. He introduces you to his sister, and you two start helping her pack what remains. Most of what they're bringing are several research papers, though you can't know what's written on them.`

> 			`	Finally, you start heading back to your ship. Noticing your interest in taking in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor has specialized in cryptocrystalline compounds, while Alituri focused her studies on planetary and cosmic conditions that lead to the formation of the gymsum caves here.`

Reviewer 5: ```suggestion
			`	Finally, you start heading back to your ship. Noticing your interest in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor specializes in cryptocrystalline compounds, while Alituri focuses her studies on planetary and cosmic conditions that lead to the formation of the gypsum caves here.`
```
`in taking in the beauty` can be simplified to `in the beauty` (personal preference). Changed verb tenses , as they're scientists, not ex-scientists. Assumed `gypsum`, but is it `gymsum`?

> 			`	Finally, you start heading back to your ship. Noticing your interest in taking in the beauty of the gypsum caves, the two Saryd researchers slow down their pace, and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor has specialized in cryptocrystalline compounds, while Alituri focused her studies on planetary and cosmic conditions that lead to the formation of the gymsum caves here.`

Reviewer 4: ```suggestion
			`	Finally, you start heading back to your ship. Noticing your interest in taking in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor has specialized in cryptocrystalline compounds, while Alituri focused her studies on planetary and cosmic conditions that lead to the formation of the gymsum caves here.`
```

> 			`	With the little "tour" over, when you three reach your ship, you show them to their bunks, and prepare to head back to <planet>.`

Reviewer 5: ```suggestion
			`	With the "tour" over, you three reach your ship. You show them to their bunks and prepare to head back to <planet>.`
```
Various edits and shortened a long sentence (personal preference). Double predicate; no comma.

				accept

mission "Coalition Yottrite 6"
	name "Master Shipwright"

> 	description "Head to <destination> to pick up a member of House Mallob, so that they may help in trying to apply Yottrite to an alloy."

Reviewer 5: ```suggestion
	description "Head to <destination> to pick up a member of House Mallob so that they may help in trying to apply Yottrite to an alloy."
```
Independent clause followed by dependent clause; no comma.

	landing
	to offer
		has "Coalition Yottrite 5: done"
	source "Delve of Bloptab"
	destination "Market of Gupta"
	on offer
		conversation

> 			`Lugglop is waiting for you when you land, and greets the two researchers as you help them get their things out of the ship.`

Reviewer 5: ```suggestion
			`Lugglop is waiting for you when you land and greets the two researchers as you help them get their things out of the ship.`
```
Double predicate; no comma.

> 			`	He points them to a hotel, where he had a room prepared for them, and asks you to follow him to his office again.`

Reviewer 5: ```suggestion
			`	He points them to their accommodations and asks you to follow him to his office again.`
```
OR
```suggestion
			`	He points them to a hotel where he had a room prepared for them and asks you to follow him to his office again.`
```
Thought this extra information wasn't substantial enough to justify its inclusion, but up to you (personal preference). Dependent clause following an independent clause; no comma.

			`	"Filled to the brim with cutting edge equipment, our facilities here and on the neighbor planet are. Alas, venture much into the theoretical field needed to study such rare elements, our personnel here does not."`

> 			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`

Reviewer 3: ```suggestion
			`	He shows you to his map again, this time pointing to a nearby Arach world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
```

Author: the map label spells out `Arachi`, so when referring to one of their worlds being shown in a map I'd say using `Arachi` is better.

on a similar not Wizrad questioned in PMs whether Kimek or Kimeks was plural, I was using the latter but the examples in the game currently use the former, and given the label does too I'll be changing mine to fit

Reviewer 3: Still, it's "human world", "wanderer world" etc, not "humans world". Becaus of that, I'm still convinced it should be Arach world

Author: I'd argue that's because both of those follow the conventional "plural = word with s at the end" for when referring to their species as a group, while with the Arachi it's different, so not feeling it much for this change

Reviewer 3: It doesn't matter how the plural of a word is, the rule is always the same

Author: just that `Arachi world` reads better than `Arach world` to me

> 			`	You're about to head back to your ship, when he hands you 200,000 credits. "Nearly forget to pay you for this last service, I did."`

Reviewer 6: line 335 : forget --> forgot 
```suggestion
			`	You're about to head back to your ship, when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`
```

> 			`	You're about to head back to your ship, when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`

Reviewer 5: ```suggestion
			`	You're about to head back to your ship when he hands you <payment>. "Nearly forgot to pay you for this last service, I did."`
```
Can't nail the exact reason the comma shouldn't be here, but I'm pretty much 100% sure it shouldn't.

Author: already addressed the payment thing in previous comments

				accept
	on accept
		payment 200000

mission "Coalition Yottrite 7"
	name "Master Shipwright"
	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."
	passengers 1
	landing
	to offer
		has "Coalition Yottrite 6: done"
	source "Market of Gupta"
	destination "Delve of Bloptab"
	on offer
		conversation

> 			`You're immediately hailed by House Mallob when your ship is descending the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive castle-like structure, which appears to be made out of a similar material to that of Arach ships, at the base of a mountain.`

Reviewer 5: ```suggestion
			`You're immediately hailed by House Mallob when your ship enters the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive, castle-like structure at the base of a mountain. It appears to be made out of a similar material to that of Arach ships.`
```
Long sentence (personal preference). Two adjectives describing `structure`, need a comma between them. Moved `at the base of a mountain` closer to what it describes for clarity.

Author: did my change help clarify, or does it need to be changed still?

> 			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`

Reviewer 5: ```suggestion
			`	Several buildings surround a dedicated docking area where other ships land close to your own. You're almost certain the building closest to where you've landed is an outfitter. It looks like a small village that's been converted into an outdoor shopping mall.`
```
It feels like there are a lot of different descriptors of the area strung into compound sentences for no other reason than to make compound sentences. Suggested a rewrite above. If you choose not to use mine, I'd recommend you come up with something else.

Author: redid

> 			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels that of a small village turned into an outdoors shopping mall.`

Reviewer 4: ```suggestion
			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`
```

> 			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, and pondering if you would comfortably fit inside one.`

Reviewer 6: line 355 : and pondering --> while pondering
```suggestion
			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`
```

> 			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`

Reviewer 5: ```suggestion
			`	You're greeted by an Arach who arrived in what looks like a car (though you gave up on trying to guess the exact purpose of each pipe and plating as well as why some wheels aren't touching the ground). You wonder if you would comfortably fit inside one.`
```
Not a big fan of the current writing of this description. As above, I've suggested a fix, but I would recommend a rewriting if you decide not to use mine.

Author: also redid

> 			`	The Arach takes a translation device out of his pocket, and formally greets you, "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`

Reviewer 5: ```suggestion
			`	The Arach takes a translation device out of his pocket and formally greets you, "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
```
Double predicate; no comma.

			choice
				`	"Are you the shipwright I'll be taking to Lugglop?"`
					goto servant
				`	"What's with this place? Why are there so many buildings all over?"`
					goto property
			label servant

> 			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as chauffeur, I work."`

Reviewer 5: ```suggestion
			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as a chauffeur, I work."`
```

				goto mistress
			label property

> 			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`

Reviewer 3: ```suggestion
			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arach ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
```

> 			`	"No mere mansion, the private property of House Mallob is," he says, as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`

Reviewer 5: ```suggestion
			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
```
Independent clause followed by dependent clause; no comma.

			label mistress

> 			`	He asks that you go inside the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`

Reviewer 5: ```suggestion
			`	He asks that you enter the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`
```
(Personal preference.)

> 			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`

Reviewer 3: ```suggestion
			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arach vessel, though you still don't know what the extra wheels are for.`
```

Author: I rewrote the part since it was frankly a mess, so this section is no longer an issue since it doesn't exist. if you spot any errors on the new stuff do point it out

> 			`	The short trip is uneventful, you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`

Reviewer 4: ```suggestion
			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`
```

> 			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`

Reviewer 5: ```suggestion
			`	You manage to take a seat without much discomfort in the Arachi vessel (fortunately, it's a short trip), though you still don't know what the extra wheels are for.`
```
Suggested (personal preference). If you prefer not to, I would still recommend rewording `with you managing to have a seat without much discomfort in the Arachi vessel`.

Author: also also redid

> 			`	The entrance to the mansion isn't as castle-like as the walls, though it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`

Reviewer 5: ```suggestion
			`	The entrance to the mansion isn't as castle-like as the walls, though it all seems strangely pompous. Up close, it almost looks like the several parts and pods of an Arach Hulk (or two) were crumpled together into one single building.`
```
Various edits to improve flow (personal preference).

> 			`	He guides you through the mansion. You pass through several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students who came here to study with the shipwrights of House Mallob.`

Reviewer 2: ```suggestion
			`	He guides you through the mansion. You pass by several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students who came here to study with the shipwrights of House Mallob.`
```

> 			`	He guides you through the mansion, as you pass through several other servants, some rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students on the way, here to study with the shipwrights of House Mallob.`

Reviewer 5: ```suggestion
			`	He guides you through the mansion. You pass through several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students here to study with the shipwrights of House Mallob.`
```
Various edits to fix multiple grammar issues. If you want me to elaborate, just let me know.

Author: didn't follow it exactly on that last part, any new issues from that?

> 			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside You enter, only to see a Kimek glancing out the window.`

Reviewer 4: ```suggestion
			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
```

			choice
				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
				`	"A Kimek? I thought this was an Arach House."`

> 			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`

Reviewer 2: ```suggestion
			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student at the university here, she was."`
```

> 			`	The butler ignores what you said, and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`

Reviewer 5: ```suggestion
			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`
```
Double predicate; no comma.

> 			`	Finally the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but as she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`

Reviewer 4: ```suggestion
			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but as she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`
```

> 			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`

Reviewer 5: ```suggestion
			`	Finally, the reason why you had to come all the way to this part of <origin> becomes clear; Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she keeps asking more and more questions ranging from how you acquired a Jump Drive to all the places you've been.`
```
Everything here is technically optional (personal preference), but I would highly recommend either these changes be made or this paragraph rewritten.

> 			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but as she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`

Reviewer 6: line 377 : but as she --> but she
```suggestion
			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`
```

			`	About an hour after the sun sets, she's finally satisfied, and after saying her goodbyes to other members of the House, you're both brought to your ship.`
				accept
	on complete
		event "yottrite research break" 165
		payment 200000
		conversation
			`As with the Saryd twins, Lugglop greets Pichiie and tells her of a hotel he prepared for her time here.`
			`	"Ready, my team is now, Captain," he says, after Pichiie leaves with her luggage. "Enrich and research samples as needed, for the next few months we will. Contact you again, I will, should the need for your services arise once again."`
			`	He hands you <payment>, and walks away.`

event "yottrite research break"

mission "Coalition Yottrite 8"
	name "One Last Sample"

> 	description "Acquire one sample of Yottrite, and deliver it to <destination>, so that Lugglop has some for his presentation to the Coalition's scientific community."

Reviewer 5: ```suggestion
	description "Acquire one sample of Yottrite and deliver it to <destination> so that Lugglop has some for his presentation to the Coalition's scientific community."
```
Double predicate and a dependent clause following independent clause; no comma for both.

	landing
	to offer
		has "event: yottrite research break"
	source

> 		government "Coalition" "Heliarch"

Reviewer 2: You could also use `near <whatever system Pelubta Station is in> 1 100`.

	destination "Pelubta Station"
	on offer
		conversation
			`When you land, you receive a message from Lugglop, the Arach who you brought several Yottrite samples to.`
			`	"Captain, some major findings, we have. Preparing a presentation about our research, to present in Ring of Wisdom, we are," he says. "Only, run out of pure samples, we have. Bring us one once more, would you?"`
			choice
				`	"Okay, I'll bring you a piece of Yottrite once again."`
				`	"Sorry, but I'm not mining that any more."`
					decline

> 			`	He thanks you, and says he and the rest of the team will be waiting in <destination>. "Moved here about halfway through the project, we did, to make use of the station's facilities."`

Reviewer 5: ```suggestion
			`	He thanks you and says he and the rest of the team will be waiting in <destination>. "Moved here about halfway through the project, we did, to make use of the station's facilities."`
```
Double predicate, no comma.

				accept
	on complete
		outfit "Yottrite" -1
		payment 220000
		conversation

> 			`Lugglop pays you <payment> as you deliver the Yottrite, accompanied by some Heliarch agent.`

Reviewer 5: ```suggestion
			`Lugglop, accompanied by a Heliarch agent, pays you <payment> as you deliver the Yottrite.`
```
(Personal preference).

> 			`	"Finish preparing our presentation, we now must. If to help us further, you wish, in the spaceport meet us you must."`

Reviewer 2: When you have short on complete conversation like this, you could just use a dialog instead.

mission "Coalition Yottrite 9"
	name "Yottrite Presentation"
	description "Transport Lugglop and his team, alongside <cargo>, to <destination>, where they will share with the Coalition's scientific community their findings on Yottrite."
	passengers 6
	cargo "Yottrite Presentation Materials" 6.14
	to offer
		has "Coalition Yottrite 8: done"
	source "Pelubta Station"
	destination "Ring of Wisdom"
	on offer
		conversation

> 			`You meet Lugglop in the spaceport and follow him to where his team is. You meet again with Nasilor, Alituri and Pichiie as the three of them oversee some workers place a few sealed metal crates inside a container and the Yottrite sample you just brought here into another.`

Reviewer 2: ```suggestion
			`You meet Lugglop in the spaceport and follow him to meet Nasilor, Alituri, and Pichiie, who are overseeing some workers place a few sealed metal crates inside a container and the Yottrite sample you just brought here into another.`
```

> 			`You meet Lugglop in the spaceport, and follow him to where his team is. You meet again with Nasilor, Alituri and Pichiie, as the three of them oversee some workers place a few sealed metal crates inside a container, and the Yottrite sample you just brought here into another.`

Reviewer 5: ```suggestion
			`You meet Lugglop in the spaceport and follow him to where his team is. You meet again with Nasilor, Alituri and Pichiie as the three of them oversee some workers place a few sealed metal crates inside a container and the Yottrite sample you just brought here into another.`
```
Double predicate, no comme (x2). Dependent clause following independent; no comma.

> 			`	They detail how they have prepared all the material they needed, and now just need a transport to <destination>.`

Reviewer 5: ```suggestion
			`	They detail how they have prepared all the material they needed and now just need a transport to <destination>.`
```
Double predicate; no comma.

> 			`	"If with us, you come, Captain, perhaps a better detailed  description of the systems where the Yottrite was found, you could provide," Nasilor says.`

Reviewer 6: fixed double space in "detailed description"
```suggestion
			`	"If with us, you come, Captain, perhaps a better detailed description of the systems where the Yottrite was found, you could provide," Nasilor says.`
```

			`	"<payment>, I would pay you," Lugglop says.`
			choice

> 				`	"Of course I'll take you there. Have that container loaded into my ship."`

Reviewer 4: ```suggestion
				`	"Of course I'll take you there. Have the containers loaded into my ship."`
```

					goto accept
				`	"What discoveries have you made about it?"`
					goto findings
			label findings
			`	"To share our findings with you, allowed we are not," Pichiie says.`
			`	"Forgive us you must, Captain, but told to keep everything a secret, we were," Lugglop continues. "Perhaps after our presentation, public our findings could be made, though."`
			label accept
			`	They all follow you to your ship after the containers are ready to be loaded.`

> 			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents are accompanying him. He introduces you to the two of them, both Kimek, though you can't quite catch their names.`

Reviewer 2: ```suggestion
			`	After they are put into your cargo, Nasilor, Alituri, and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents are accompanying him. He introduces you to the two of them, both Kimek, though you can't quite catch their names.`
```
Why don't I catch their names? If they aren't important enough to be named for the player, just don't have them be introduced.

			`	"Here to ensure the secrecy of our discoveries, they are," he says. Seeing as they are only two more passengers, you agree to carry them to <planet>, and they all board your ship.`
				accept
	on complete
		payment 310000
		conversation
			`	Immediately when you land, Lugglop gives you your payment of <payment>. You and the four researchers then follow some Heliarch agents that were waiting for you at the landing area.`

> 			`	You five are brought to an area akin to a theater, with some type of podium on the stage. The seats are all taken by Heliarch agents.`

Reviewer 5: ```suggestion
			`	You are brought to an area akin to a theater with some type of podium on the stage. The seats are all taken by Heliarch agents.`
```
`five` seemed unnecessary (personal preference). Comma unnecessary.

			`	Lugglop's team apparently will wait until the contents of the two containers are brought here, so you're asked to describe the systems with supernovae where you encountered Yottrite asteroids.`
			`	This only lasts a few minutes, as the audience doesn't seem too keen on inquiring about details, though one particular Saryd did puzzle you when they asked if the all the supernovas you visited "were natural occurrences, or artificially produced."`

> 			`	After you finish, and right as the materials for Lugglop's presentation arrives, you're escorted back to your ship by some agents.`

Reviewer 5: This ending seems a bit sudden. Maybe some expansion as to what happens to the rest of the researchers post-meeting?

Reviewer 2: Agreed on this being a rather abrupt ending.
